lua/nvim-lsp-installer/ui/display.lua

```diff
@@ -143,38 +173,40 @@ function M.new_view_only_win(name)
             end
         end
 
-        return win
+        return win_id
     end
 
     local draw = process.debounced(function(view)
-        local win = vim.fn.win_findbuf(buf)[1]
-        if not win or not vim.api.nvim_buf_is_valid(buf) then
+        -- local win = vim.fn.win_findbuf(buf_handle)[1]
+        if not win_id or not vim.api.nvim_buf_is_valid(buf_handle) then
             -- the window has been closed or the buffer is somehow no longer valid
             unsubscribe(true)
-            return log.debug { "Buffer or window is no longer valid", name, win, buf }
+            return log.debug { "Buffer or window is no longer valid", name, win_id, buf_handle }
         end
 
-        local win_width = vim.api.nvim_win_get_width(win)
+        local win_width = vim.api.nvim_win_get_width(win_id)
         local context = {
             win_width = win_width,
         }
         local output = render_node(context, view)
         local lines, virt_texts, highlights = output.lines, output.virt_texts, output.highlights
 
         vim.api.nvim_buf_clear_namespace(0, namespace, 0, -1)
-        vim.api.nvim_buf_set_option(buf, "modifiable", true)
-        vim.api.nvim_buf_set_lines(buf, 0, -1, true, lines)
-        vim.api.nvim_buf_set_option(buf, "modifiable", false)
+        vim.api.nvim_buf_set_option(buf_handle, "modifiable", true)
+        -- vim.api.nvim_buf_set_lines(buf_handle, 0, -1, true, lines)
+        vim.api.nvim_buf_set_lines(buf_handle, 0, -1, false, lines)
```

Comment:
Huh I wonder why I had put the `strict_indexing` arg to `true`, let's change to `false` :+1:

Reply:
I wasn't sure what this does, that's why I left it commented out so I can remember it 😅

Reply:
I believe it makes it so it errors if you try to set a line that doesn't yet exist in the buffer?